src/patterns/check-answers/index.md.njk

```diff
@@ -58,10 +56,10 @@ For example:
 
 <div class="govuk-!-mb-r6">
 ```html
-<a href="/renew/name/edit">Change<span class="visually-hidden"> name</span></a>
+<a href="/renew/name/edit">Change<span class="govuk-h-visually-hidden"> name</span></a>
```

Comment:
Should this be a proper example? I'm not sure it's immediately clear what the effect of this is for the reader.

Reply:
I think I might kill this whole section as it will be covered in the real example / it's not the only thing you'll need to do